index.html

```diff
@@ -137,7 +137,7 @@ <h1>
 
     <a class="anchor" id="howItWorks"></a>
     <div class="method-screen">
-      <h1>How does it <b>work?</b></h1>
+      <h1>How does it <b>works?</b></h1>
```

Comment:
That's not correct English. It's either `How does it work` or `How it works`.

Reply:
Oh right, "does" cancels present simple, forgot about that rule

Reply:
https://ell.stackexchange.com/questions/17778/how-it-works-vs-how-does-it-work